sample-service/src/main/java/uk/ac/ox/ndph/mts/sample_service/controller/MainController.java

```diff
@@ -13,6 +13,11 @@
 @RestController
 public class MainController {
 
+    private ConfigService myConfig;
```

Comment:
I think the var name should be changed myConfig -> something else . maybe configService?

Reply:
fair point although its been like this for a while and its only in the sample service so not sure we need to change?